database/redis/metric.go

```diff
@@ -106,6 +106,7 @@ func (connector *DbConnector) SaveMetrics(metrics map[string]*moira.MatchedMetri
 	ctx := connector.context
 
 	rand.Seed(time.Now().UnixNano())
+	pipe := c.TxPipeline()
```

Comment:
and changed we filters together?

Reply:
Didn't get the question

Reply:
this changes will be affected filter service or not?

Reply:
it's may me my misunderstanding...

Reply:
Yes, changes will affect filter and checker services